plugins/a11ychecker.md

```diff
@@ -161,91 +295,212 @@ If `a11ychecker_allow_decorative_images` is set to `false`, the Accessibility Ch
 
 **Possible Values:** `true`, `false`
 
-#### Example: a11ychecker_allow_decorative_images
+#### Example: Using `a11ychecker_allow_decorative_images`
 
 ```js
 tinymce.init({
-    selector: 'textarea',
-    plugins: 'a11ychecker',
-    toolbar: 'a11ycheck',
-    a11ychecker_allow_decorative_images: true
+  selector: 'textarea',
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_allow_decorative_images: true
+});
+```
+
+### `a11ychecker_filter_issue`
+
+The `a11ychecker_filter_issue` option allows Accessibility Checker rules to be removed from the Accessibility Checker report using a callback function. This option is a flexible alternative to the `a11ychecker_ignored_rules` option.
+
+The function will be passed the result object of the [`getReport()` API](#getreport). To filter issues from the Accessibility Checker report, the callback needs to return `false` for the issue.
```

Comment:
"result object"?

Reply:
```suggestion
The function will be passed the result object of the [`getReport()` API](#getreport). To filter an issue from the Accessibility Checker report, the callback needs to return `false` for that particular issue.
```
maybe, for clarity?

Reply:
this also runs when you open the dialog too (since it calls the `getReport` API)